tests/v2/waku_archive/test_waku_archive.nim

```diff
@@ -467,7 +440,6 @@ procSuite "Waku Archive - find messages":
       response.cursor.isNone()
 
   test "handle temporal history query with a valid time window":
-    ## Given
```

Comment:
Why remove this?

Reply:
ah yes, this is totally beyond the scope of this PR. I will set it back